src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -362,19 +364,21 @@ private void prepareStatements() {
     insertRepairSegmentPrepStmt = session
         .prepare(
             "INSERT INTO repair_run"
-                + "(id,segment_id,repair_unit_id,start_token,end_token,segment_state,fail_count, token_ranges)"
+                + "(id,segment_id,repair_unit_id,start_token,end_token,segment_active,segment_state,"
```

Comment:
This one and next two statements are broken. Extra question mark is required in values section

Reply:
Thank you very much for figuring this out!

I just force-pushed the fix for the three queries.

Reply:
I think we did it. version 2.0.2 start on clear cassandra keyspace normaly
```
Mar 24 23:29:30 zx-prod-tools-cassandra-reaper-1.example.org cassandra-reaper[3901]: WARN   [2020-03-24 23:29:30,228] [main] i.c.ReaperApplication - Reaper is ready to get things done!
```
Now we gonna try 2.0.3 version with this fix but i think it's working fine

Reply:
@smarsching you are welcome!

Reply:
version 2.0.3 works great
Mar 24 23:54:53 zx-prod-tools-cassandra-reaper-1.example.org cassandra-reaper[5998]: WARN   [2020-03-24 23:54:53,065] [main] i.c.ReaperApplication - Reaper is ready to get things done!